frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts

```diff
@@ -38,34 +38,16 @@ import {HalResourceNotificationService} from "core-app/modules/hal/services/hal-
 import {NgSelectComponent} from "@ng-select/ng-select";
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
+import {UserResource} from "core-app/modules/hal/resources/user-resource";
 
 export const usersAutocompleterSelector = 'user-autocompleter';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/18f66b2ef9cf25730db4c2a6cda0e5264272cbca/frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts#L44"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>